transform/models/marts/fact_commits_monthly.sql

```diff
@@ -0,0 +1,15 @@
+with monthly_commits as (
+  select 
+    substr(c.date, 5,2) as date_month, 
```

Comment:
Since we don't want to convert date as string, we don't need to use substr and can use date_trunc instead.